src/runtime/test/suggestion-composer-tests.ts

```diff
@@ -9,96 +9,105 @@
  */
 
 import {assert} from './chai-web.js';
+
+import {Arc} from '../arc.js';
+import {SlotComposer} from '../slot-composer.js';
 import {SuggestionComposer} from '../suggestion-composer.js';
-import {FakeSlotComposer} from '../testing/fake-slot-composer.js';
+import {MockSlotComposer} from '../testing/mock-slot-composer.js';
+import {MockSuggestDomConsumer} from '../testing/mock-suggest-dom-consumer.js';
 import {TestHelper} from '../testing/test-helper.js';
 
 class TestSuggestionComposer extends SuggestionComposer {
-  constructor() {
-    super(null, new FakeSlotComposer({containers: {suggestions: {}}}));
-    this.suggestions = [];
-    this.updatesCount = 0;
-    this.updateResolve = null;
-  }
-
-  setSuggestions(suggestions) {
-    this.suggestions = suggestions;
+  get suggestConsumers() {
+    return this._suggestConsumers;
   }
 }
 
-describe('suggestion composer', function() {
-  it('sets suggestions', async () => {
-    const suggestionComposer = new TestSuggestionComposer();
-    assert.isEmpty(suggestionComposer.suggestions);
-
-    // Sets suggestions
-    await suggestionComposer.setSuggestions([1, 2, 3]);
-    assert.lengthOf(suggestionComposer.suggestions, 3);
-
-    await suggestionComposer.setSuggestions([4, 5]);
-    assert.lengthOf(suggestionComposer.suggestions, 2);
-
-    await suggestionComposer.setSuggestions([6, 7, 8]);
-    await suggestionComposer.setSuggestions([]);
-    assert.isEmpty(suggestionComposer.suggestions);
-  });
-
+describe('suggestion composer', () => {
```

Comment:
just curios, did you delete this test case on purpose?

Reply:
See above.

Yes, it doesn't really do anything other than testing the test class setSuggestions method, which doesn't do anything special.